src/constants/portuguese.ts

```diff
@@ -43,6 +43,16 @@ export const portugueseTextMap = {
       `Navegar para a página ${to}`
     ),
   },
+  personCard: {
+    name: 'Nome: ',
+    cpf: 'CPF: ',
+    birthDate: 'Nascimento: ',
+    civilState: 'Estado Civil: ',
+    nis: 'NIS: ',
+    phone: 'Telefone: ',
+    email: 'Email: ',
+    address: 'Endereço: ',
```

Comment:
Espaço é estilo, não tradução. Como vi no componente que fica realmente mais acessível utilizar um `<strong>` ao redor desses títulos, não temos como utilizar o método funcional passando parâmetro, mas o espaço não é importante aqui.
```suggestion
    name: 'Nome:',
    cpf: 'CPF:',
    birthDate: 'Nascimento:',
    civilState: 'Estado Civil:',
    nis: 'NIS:',
    phone: 'Telefone:',
    email: 'Email:',
    address: 'Endereço:',
```